### PR TITLE
Added discount management

### PR DESCRIPTION
**Pull request by mingazhev**

### CODE REVIEW

File: Api/AdministratorServices/AgencyDiscountManagementService.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                .Check(_ => Validate(discountInfo))
+                .Map(d => Update(d, discount =>
+                {
+                    discount.DiscountPercent = discountInfo.DiscountPercent;
+                    discount.Description = discountInfo.Description;
+                }));
+        }
+
+
+        private async Task<Result<Discount>> Get(int agencyId, int discountId)
```
Comment (DFSko) [inline Api/AdministratorServices/AgencyDiscountManagementService.cs:74]: why is id needed here?

Comment (mingazhev) [reply Api/AdministratorServices/AgencyDiscountManagementService.cs:74]: not needed if being strict, but for meaningful routes including an agency

Comment (Raibo) [reply Api/AdministratorServices/AgencyDiscountManagementService.cs:74]: as an option, we could include agencyId into the route, but not take as a parameter further, leavling it unused in the controller

Comment (mingazhev) [reply Api/AdministratorServices/AgencyDiscountManagementService.cs:74]: I'm trying to avoid unused vars

**Comment by mingazhev** [comment]
Another approach chosen

**Pull request closed without merging**